cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,20 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
```

Comment:
∃ a constant in cms/djangoapps/contentstore/views/component.py as follows:
SPLIT_TEST_COMPONENT_TYPE = 'split_test' - I would use this reference